src/loader.rs

```diff
@@ -134,5 +134,15 @@ impl_extra_handler_builder! {
     get_chat_name => GetChatNameHandler
     send_im => SendIMHandler
     chat_send => ChatSendHandler
-    get_cb_alias => GetChatBuddyAlias
+}
+
+// get_cb_alias is only available in Purple 2.14
+#[cfg(libpurple2_14)]
+impl<T: crate::purple::handlers::traits::GetChatBuddyAlias> RegisterContext<T> {
```

Comment:
Can't this use the macro `impl_extra_handler_builder!`?

Otherwise I think it can use a trick like:

```rust
mod libpurple2_14 {
  impl_extra_handler_builder! {
    get_cb_alias => GetChatBuddyAlias
  }
}

#[cfg(libpurple2_14)]
pub use libpurple2_14::*;
```

Reply:
Oooh, yes, that works really well :thumbsup: